MovieApp/Coordinator/MainRouter.swift

```diff
@@ -0,0 +1,53 @@
+import UIKit
+
+class MainRouter: RouterProtocol {
+
+    private let appDependencies = AppDependencies()
+
+    var navigationController: UINavigationController
+
+    init(navigationController: UINavigationController) {
+        self.navigationController = navigationController
+    }
+
+    func start(in window: UIWindow) {
+        navigationController.navigationBar.isHidden = true
+
+        window.rootViewController = navigationController
+        window.makeKeyAndVisible()
+
+        let tabBarController = makeUITabBarController()
```

Comment:
You should create a `showMainAppScreen()` func and call it from here. Also, that func should setVC, and not push it since it is a main screen in the app.

Reply:
`setViewControllers` is simply preferred when setting up the starting screen of the app?
From what I gathered in this situation `setViewControllers` and `pushViewController` would be equivalent in this situation since the navigation stack is empty and we're putting only one view controller on it

Reply:
set is preffered when you want to ensure that VC is first and only in stack